src/main/menus/app.js

```diff
@@ -193,7 +193,12 @@ function createTemplate(config) {
     })(),
     click(item, focusedWindow) {
       if (focusedWindow) {
-        focusedWindow.toggleDevTools({mode: 'detach'});
+        // toggledevtools opens it in the last known position, so sometimes it goes below the browserview
+        if (focusedWindow.isDevToolsOpened()) {
```

Comment:
Might be misunderstanding, but are we opening the dev tools only if they're already opened?

Reply:
no, you are right, tested it, but linter was complaining about negative conditions and I failed to copy properly 🤦